AdminCore.DataETL/TypeConverters/ListToStringConverter.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+
+namespace AdminCore.Common.Attributes.DataEtl
+{
+    /// <summary>
+    /// Converts a type to a string using the ToString() method.
+    /// </summary>
+    public class ListToStringConverter<T> : ITypeConverter
+    {
+        private string _defaultIfNull = "Empty";
+
+        public object ConvertTo(object value)
+        {
+            if (value == null)
```

Comment:
if (value == null || !value.Count.Any()) {
return _defaultIfEmpty;
}

Reply:
I haven't combined theses two statements into the one conditional because I think I need to retain the usage of the object type when ConvertTo is called in CsvChoEtlAdapter. 
So I can't reference Count until I've checked for null then cast to List.

What do you think?

I did make a change further down to not use Count.
```
if (!list.Any())
            {
                return _defaultIfNull;
            }
```

Reply:
I pushed up another commit for that csv export PR. I realised that I was already doing type conversion in the automapper mapping, so i did it all in there and removed the TypeConverter attributes. I think it's maybe better to offload it onto an AutoMapper profile where you can be specific about casts rather than the generic object casting.